__tests__/app.test.js

```diff
@@ -420,6 +420,84 @@ describe('CORE: GET /api/articles/:article_id/comments', () => {
     })
 })
 
+describe('CORE: POST /api/articles/:article_id/comments', () => {
+    test('returns a status code of 201 and an object', () => {
+        const newComment = { username: "icellusedkars", body:"Yes, if you stare at the wall like a cat, that means you're a cat"};
+        return request(app)
+        .post('/api/articles/11/comments')
+        .send(newComment)
+        .expect(201)
+        .then((response) => {
+            const bodyObj = response.body;
+            expect(typeof bodyObj).toBe('object');
+        })
+    })
+    test('returns a posted_comments object with the correct keys and value types when given the correct username and article_id', () => {
+        const newComment = { username: "icellusedkars", body: "Yes, if you stare at the wall like a cat, that means you're a cat"};
+        return request(app)
+        .post('/api/articles/11/comments')
+        .send(newComment)
+        .expect(201)
+        .then((response) => {
+            const posted_comment = response.body.posted_comment;
+            const exampleCommentObj = {
+                "comment_id": expect.any(Number),
+                "votes": expect.any(Number),
+                "created_at": expect.any(String),
+                "author": expect.any(String),
+                "body": expect.any(String),
+                "article_id": expect.any(Number)
+                }
+            
+            expect(posted_comment).toMatchObject(exampleCommentObj);
+        })
+    })
+    test('returns a status of 400 and a customer error object of {status: 400, msg: "Invalid article_id" } when using an article id that does not exist', () => {
+        const newComment = { username: "icellusedkars", body: "Yes, if you stare at the wall like a cat, that means you're a cat"};
+        return request(app)
+        .post('/api/articles/0/comments')
+        .send(newComment)
+        .expect(400)
+        .then((response) => {
+            const error = response.body;
+            expect(error.msg).toBe('Invalid article_id');
+        })
+    })
+    test('returns a status of 400 and a custom error object of {status:400, msg: "Invalid Comment"} when provided an invalid comment object', () => {
```

Comment:
might want to be specific to sya why this object is invalid i.e no body property so: returns a status of 400 and a message of "Invalid Comment" when body property is not provided